benchmarks/bench_methods_run.py

```diff
@@ -21,9 +20,6 @@
 import scipy.sparse as sp
 from joblib import Parallel, delayed
 
-import alphacsc.other.heide_csc as CSC
```

Comment:
Either keep this method in the comparison or remove the file from `other`.
I would say remove it.

Reply:
Removed